### PR TITLE
Binding desc 5/5

### PR DESCRIPTION
**Pull request by ElianaDalSasso**
Dear all,
these are the last descriptions of the bindings of Juel-Jensen manuscripts. 
I decided to describe the condition of manuscripts composed of several msPart (f. 30, b. 6, b. 3) under a general element condition because the descriptions refer to the manuscript as a whole. In the single msParts, I only left a comment on the state of preservation (good/deficient).
On the contrary, I kept separated the descriptions of holes and tears on the writing support (except in b. 3 where the different msPart are not encoded yet).
Let me know what you think!

In the case you agree, I have to check if this approach has been applied consistently.

Thank you!
Eliana

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! I agree with your general proposal regarding the distribution of information in composite manuscripts. Just some minimal remarks here

**Review by thea-m — APPROVED** [review]

File: OxfordBodleian/Juel-Jensen/BDLaethf30.xml [hunk 1]
```diff
@@ -136,24 +136,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                                    and at the end. The sewing has become partly undone. 
+                                <condition key="deficient">
+                                  The manuscript is in a poor state of preservation.
+                                  <locus target="#i"/> is much less broad than the other leaves, but this might be its original state.
+                                  The manuscript is damaged by humidity, in particular at the beginning
+                                  and at the end. The lower margin is darkened by use. The ink is occasionally smudged, e.g. <locus target="#41v #104v"/>.
+                                  The leaves present stains of various origins, uneven deformations, tears, and losses,
+                                  which sometimes result in more or less extensive losses of text, e.g. <locus target="#55 #150 #152"/>.
+                                  Dark red stains on <locus from="20" to="23"/> and <locus target="#115"/>.
+                                  <!--the stains could be the result of oxidation of the parchment support
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethf30.xml:147]: In my view we can also incorporate this comment in the text, for example "Dark red stains on ..., possibly due to oxidation of the support"

File: OxfordBodleian/Juel-Jensen/BDLaethf30.xml [hunk 2]
```diff
@@ -170,59 +186,99 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                           <material key="wood"/>
-                        </decoNote>
-                        leaf string marker f.  39, 101, 104
+
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="true Ethiopian">
+                              <p>
+                                The manuscript was repaired several times, as testify the various threads visible on the spine and holes drilled through the boards.
+                                However, the materials and the techniques used are compatible with those of the Ethiopian binding tradition.
+                                The boards and the cover are possibly contemporary with most of the manuscript content.</p>
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethf30.xml:217]: I would say "possibly contemporary with the main texts" (which recalls the hierarchy adopted in the textual part)

File: OxfordBodleian/Juel-Jensen/BDLaethg33.xml [hunk 3]
```diff
@@ -496,31 +499,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                                <decoNote xml:id="b7" type="Other">Blue <term key="leafTabMark">leaf tab markers</term> are inserted in the outer margins of 
+                              <binding contemporary="true Ethiopian">
+                                <p>The threads used to repair the sewing and the chemise are later additions to the original binding.
+                                  However, most of the binding (the boards and the sewing thread) are likely contemporary to the manuscript content.</p>
+                                  <decoNote xml:id="b1" type="bindingMaterial">
+                                    The book is bound in <material key="wood">wooden</material> boards
+                                    enclosed in a <material key="textile">textile</material> chemise.
+                                  </decoNote>
+                                  <decoNote xml:id="b2" type="Boards">
+                                    The upper board is narrower than the book block. It has broken and lost part of the outer edge.
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg33.xml:510]: has been broken o is broken?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Eliana for your very rich descriptions! I just have minimal observations.
I agree with your subdivision for the composite MSS: it is the best way to discriminate between features of the whole codex and features of the single parts.

**Review by MassimoVilla — APPROVED** [review]

**Review by gnizla — APPROVED** [review]

**Pull request merged**